projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -337,6 +337,16 @@ public static function get_jetpack_gutenberg_extensions_allowed_list() {
 		return self::get_extensions_preset_for_variation( $preset_extensions_manifest, $blocks_variation );
 	}
 
+	/**
+	 * Check if the given extension is available.
+	 *
+	 * @param string $extension The extension to check.
+	 * @return bool             True if the extension is beta. Otherwise, false.
+	 */
+	public static function is_extension_available( $extension ) {
```

Comment:
Do we need this here instead of the existing `is_available` method? If so, maybe we should extend both methods' descriptions to explain how different they are, and which one to use when.

Reply:
`is_available()` I think checks whether the extension is available in terms of the site plan and products. etc. AFAIK, it doesn't pay attention to the extension type (`beta`, `experimental`, ...)

```php
$is_available = \Jetpack_Gutenberg::is_available( 'videopress/video' );
```

```
[24-Aug-2022 11:50:07 UTC] $is_available:
```

Inspecting the self::$availability array, it's empty. `¯\_(ツ)_/¯`

```
[24-Aug-2022 11:50:07 UTC] self::$availability: Array
(
)
```

Anyway, the purpose of this PR is to be able to register the block depending on the extension type.

Reply:
I've removed the method from the Jetpack_Gutenberg class for now.